crates/rspack_plugin_css/tests/fixtures/webpack/at-charset/expected/main.css

```diff
@@ -19,6 +19,11 @@
   background: green;
 }
 @charset "utf-8";
+;
+;
+;
+;
+;
```

Comment:
+1 here

Reply:
This always cause by https://github.com/swc-project/swc/issues/7584, the end of decl is not correct.